source/neuropod/backends/tensorflow/tf_backend.cc

```diff
@@ -152,47 +145,40 @@ TensorflowNeuropodBackend::TensorflowNeuropodBackend(const std::string &neuropod
     }
 }
 
-void TensorflowNeuropodBackend::load_model_internal()
+void TensorflowNeuropodBackend::load_saved_model()
 {
-    // Load custom ops (if any)
-    for (const auto &item : model_config_->custom_ops)
-    {
-        const auto path = "0/ops/" + item;
-        const auto hash = loader_->get_hash_for_file(path);
+    tensorflow::SavedModelBundle bundle;
+    constexpr char               kSavedModelTagServe[] = "serve";
+    check_tf_status(tensorflow::LoadSavedModel(
+        get_tf_opts(options_), {}, loader_->get_file_path("0/data/savedmodel"), {kSavedModelTagServe}, &bundle));
 
-        // Don't load a custom op if we've already loaded it
-        std::lock_guard<std::mutex> lock(loaded_op_mutex);
-        if (loaded_op_hashes.count(hash) == 0)
-        {
-            if (dlopen(loader_->get_file_path(path).c_str(), RTLD_NOW) == nullptr)
-            {
-                const auto err = dlerror();
-                if (err == nullptr)
-                {
-                    NEUROPOD_ERROR("Failed to load custom op. dlopen failed but no error was available");
-                }
-                else
-                {
-                    NEUROPOD_ERROR("Failed to load custom op. Error from dlopen: {}", err);
-                }
-            }
+    session_ = std::move(bundle.session);
 
-            loaded_op_hashes.insert(hash);
-        }
+    // Get signatures
```

Comment:
It's not clear what these signatures represent, can you expand the comment to include an explanation?

Reply:
Will do. I'll also add a reference to the TF docs